external/model-preparation-algorithm/configs/detection/configuration.yaml

```diff
@@ -119,6 +119,37 @@ learning_parameters:
     value: 0
     visible_in_ui: true
     warning: null
+  enable_early_stopping:
+    affects_outcome_of: TRAINING
+    default_value: true
+    description: Adaptive early exit from training when accuracy isn't changed or decreased for several epochs.
```

Comment:
I'm confused with "Adaptive".. Does that mean early stopping by adaptive interval?